src/mozxchannel/git/pull_request.py

```diff
@@ -16,4 +16,11 @@ def create(target, target_branch, *, title, message, branch=None):
     env = os.environ.copy()
     env["EDITOR"] = "true"
     cmd = ["git", "-C", target, "pull-request", "--target-branch", target_branch, "--title", title, "--message", message]
-    subprocess.run(cmd, env=env, check=True)
+    try:
+        subprocess.run(cmd, env=env, check=True)
+    except SystemExit as exc:
```

Comment:
I think we should catch `subprocess.CalledProcessError` and look at `exc.returncode`?

Reply:
done.